server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/ApplyTemplateAction.java

```diff
@@ -64,7 +64,7 @@ public void define(WebService.NewController context) {
       .setHandler(this);
 
     createTemplateParameters(action);
-    createProjectParameter(action);
+    createProjectParameters(action);
```

Comment:
It would have been nice to have this renaming done in another commit, as it's a "side" modification.
Moving it in another commit makes commits clearer